docs/options.md

```diff
@@ -177,7 +177,9 @@ A shell command to run before building the wheel. This option allows you to run
 
 If dependencies are required to build your wheel (for example if you include a header from a Python module), set this to `pip install .`, and the dependencies will be installed automatically by pip. However, this means your package will be built twice - if your package takes a long time to build, you might wish to manually list the dependencies here instead.
 
-The active Python binary can be accessed using `python`, and pip with `pip`; `cibuildwheel` makes sure the right version of Python and pip will be executed. `{project}` can be used as a placeholder for the absolute path to the project's root.
+The active Python binary can be accessed using `python`, and pip with `pip`; `cibuildwheel` makes sure the right version of Python and pip will be executed. `{project}` can be used as a placeholder for the absolute path to the project's root and will be replaced by `cibuildwheel`.
+
+On Linux and macOS, the command is run in a shell, so you can write things like `cmd1 && cmd2`.
```

Comment:
And on Windows?

Reply:
To be honest I have no idea, I've just copy-pasted this from the `CIBW_TEST_COMMAND` documentation.

Reply:
Hmmm, OK. I guess I was/am also just confused by changing this in the docs for CIBW_BEFORE_BUILD, while this PR is not touching that part of the code.